src/spectrum.jl

```diff
@@ -144,6 +159,142 @@ function _spectrum(
     return spec
 end
 
+function _spectrum(
+    L::QuantumObject{SuperOperator},
+    ωlist::AbstractVector,
+    A::QuantumObject{Operator},
+    B::QuantumObject{Operator},
+    solver::Lanczos;
+)
+    check_dimensions(L, A, B)
+
+    # Define type shortcuts
+    fT = _FType(L)
+    cT = _CType(L)
+
+    # Calculate |v₁> = B|ρss>
+    ρss =
+        solver.steadystate_solver === nothing ? mat2vec(steadystate(L)) :
```

Comment:
`isnothing` should make it more compact

Reply:
I was thinking that, instead of making `steadystate_solver` `Nothing` as default, one could choose it as the direct one as default (as in steady state), and then avoid using this if condition

Reply:
> I was thinking that, instead of making `steadystate_solver` `Nothing` as default, one could choose it as the direct one as default (as in steady state), and then avoid using this if condition

I wanted to provide a more predictable and uniform behavior for the user, since `nothing` will use whatever the default solver is in `spectrum`; changing that default will also change this one, so maybe it's easier and one doesn't have to track default solvers across different functions. But as you prefer, really; what's @ytdHuang preference on this?